Update 04.protein-extraction.md (#50)
jessegmeyerlab/proteomics-tutorial: content/04.protein-extraction.md

#### Cell lysis 
One typical lysis buffer is 8 M urea in 100 mM Tris, pH 8.5; the pH based on optimum trypsin activity [@PMID:25664860]
Small mammalian cell pellets and exosomes will lyse almost instantly upon addition denaturing buffer. 

> If non-denaturing conditions are desired, osmotic swelling and subsequent shearing or sonication can be applied [@DOI: 10.1080/10826068.2020.1728696].

Reviewer: ```suggestion
If non-denaturing conditions are desired, osmotic swelling and subsequent shearing or sonication can be applied [@DOI:10.1080/10826068.2020.1728696].
```

Typically, disulfide bonds in proteins are reduced and alkylated prior to proteolysis in order to disrupt structures and simplify peptide analysis. 
This allows better access to all residues during proteolysis and removes the crosslinked peptides created by S-S inter peptide linkages. 
There are a variety of reagent options for these steps. 

> For reduction, the typical agents used are 5-15 mM concentration of TCEP-HCl/DTT/2BME.

Reviewer: ```suggestion
For reduction, the typical agents used are 5-15 mM concentration of tris(2-carboxyethyl)phosphine hydrochloride (TCEP-HCl), dithiothreitol (DTT), or 2-mercaptoethanol (2BME).
```

> TCEP-HCl is an efficient alkylation agent, but it also significantly lowers sample pH, which can be abated by increasing sample buffer concentration or resuspending TCEP-HCl in an appropriate buffer system (i.e 1M HEPES pH 7.5).      

Reviewer: ```suggestion
TCEP-HCl is an efficient reducing agent, but it also significantly lowers sample pH, which can be abated by increasing sample buffer concentration or resuspending TCEP-HCl in an appropriate buffer system (i.e 1M HEPES pH 7.5).      
```

> For the following alkylation step, a slightly higher 10-20mM concentration, alkylating agent such as chloroacetamide/iodoacetamide, n-ethyl maleimide can be used [PMID:29019370; @PMID:15351294; @PMID:28539326]. 

Reviewer: ```suggestion
Following the reducing step, a slightly higher 10-20mM concentration of alkylating agent such as chloroacetamide/iodoacetamide or n-ethyl maleimide is used to cap the free thiols [PMID:29019370; @PMID:15351294; @PMID:28539326]. 
```

> It is also possible to alklylate free cysteines with one reagent, reduce di-sulfide bonds (or other cysteine modifications) and alkylate with a different reagent in order to monitor which residues are linked/modified in a protein [@PMID:32132231; @PMID:28445428; @PMID:23074338]. 

Reviewer: ```suggestion
In order to monitor which cysteine residues are linked or modified in a protein, it is also possible to alkylate free cysteines with one reagent, reduce di-sulfide bonds (or other cysteine modifications) and alkylate with a different reagent [@PMID:32132231; @PMID:28445428; @PMID:23074338]. 
```

> Reduction reactions are generally carried out in the dark at room temperature to avoid excessive off-target alkylation.

Reviewer: ```suggestion
Alkylation reactions are generally carried out in the dark at room temperature to avoid excessive off-target alkylation of other amino acids.
```

Pellets should be washed with organic solvent for complete removal especially of detergents. 
Alternatively, solid phase based digestion methods such as S-trap [@PMID:33750040], FASP [@PMID:19377485; @PMID:30259475], SP3 [@PMID:3117935; @PMID:28948796] and on column/bead can allow for proteins to be applied to a solid phase and buffers removed prior to proteolysis [@PMID: 29754492]. 
Specialty detergent removal columns exist (Pierce/Thermo Fisher Scientific) but add expense and time consuming steps to the process. 

> Relatively low concentrations of detergent (i.e. 1% DOC) or chaotropes (i.e. 1M urea) are compatible with digestion by trypsin/Lys-C and can be achieved by diluting the sample in appropriate buffer (i.e. 100 mM ammoniumbicarbonate, pH 8.5). 

Reviewer: ```suggestion
Relatively low concentrations of specific detergents, such as 1% deoxycholate (DOC), or chaotropes (i.e. 1M urea) are compatible with proteolysis by trypsin/Lys-C.
Often proteolysis-compatible concentrations of these detergents and chaotropes are achieved by diluting the sample in appropriate buffer (i.e. 100 mM ammonium bicarbonate, pH 8.5) after cell or tissue lysis in a higher concentration. 
```
